epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -603,6 +605,36 @@ private MethodSpec generateSetter(ClassToGenerateInfo helperClass, AttributeInfo
         .build();
   }
 
+  private MethodSpec generateReset(ClassToGenerateInfo helperClass) {
+    Builder builder = MethodSpec.methodBuilder("reset")
```

Comment:
Can we actually have this return the type of the generated class? That way you could chain it
```
model.reset().value(...).value2(...)
```

This could be done in another PR, but we'd have to do it before releasing a next version